include/sound/hda_register.h

```diff
@@ -264,6 +264,8 @@ enum { SDI0, SDI1, SDI2, SDI3, SDO0, SDO1, SDO2, SDO3 };
 #define AZX_REG_ML_LOUTPAY		0x20
```

Comment:
fix commit message

ALSA: hdac: clear link output stream mapping

Fix a DMA hang issue on some Intel hda devices.

The reset value for the LOSDIV register is all output streams valid.
Clear this register to invalidate non-existent streams when the bus is powered up.

Reply:
Thanks!

Reply:
> Fix a DMA hang issue on some Intel hda devices.

And maybe add some brief info how the hang happens E.g. "Fix DMA hang upon starting playback" or whatever the known use-case is. Or just "when starting DMA streams" if it is really generic.

Reply:
yes, that would help.
Also 'some' intel devices is too vague, we should be explicit on which PCH generation is impacted by this change (e.g. GLK/CML/ICL, no customer or actual product names please...)
And there should also be a statement on whether there is any adverse/side effect on other platforms (there shouldn't).

FWIW, please see the following (public) document on what maintainers will look for in a commit message:
https://docs.google.com/presentation/d/1LjLzk11tW6tBYTDy3PfWpRhZbaWSbhjWKWItfda5icU/edit#slide=id.p

Reply:
Thanks! I will  study this document.